frontend/src/components/elements/Spinner/styled-components.ts

```diff
@@ -21,14 +21,17 @@ import isPropValid from "@emotion/is-prop-valid"
 
 export const ThemedStyledSpinner = styled(StyledSpinnerNext, {
   shouldForwardProp: isPropValid,
-})(({ theme }) => {
+})(({ theme, $themeName }) => {
   return {
     marginTop: theme.spacing.none,
     marginBottom: theme.spacing.none,
     marginRight: theme.spacing.none,
     marginLeft: theme.spacing.none,
     borderColor: theme.colors.fadedText10,
-    borderTopColor: theme.colors.primary,
+    borderTopColor:
+      $themeName === "Custom Theme"
+        ? theme.colors.primary
+        : theme.colors.blue70,
```

Comment:
Note that we have an `isPresetTheme` function that can be used to do this in a slightly more robust way, so I would recommend using that and plumbing the return value as a boolean prop to the styled component.

Reply:
Oh, I didn't know we had that. Updated in [b87c19b](https://github.com/streamlit/streamlit/pull/5086/commits/b87c19b15edad1ab2db56d59bd9ae0f24d5f136e)